docker-registry-sync/README.md

```diff
@@ -50,26 +51,34 @@ stages:
       - destination: aws # key comes from registries
         repository: "simcore/services/comp_v3/sleeper" # different namespace is also allowed
         tags: ["0.0.1", "1.0.0"] # specify multiple tags
+    id: "i-am-not-required" # optional, not needed here
 
   # more advanced usage
   # pulls all sleeper tags from dockerhub and pushes to master
+  # to run this stage before another one, give it an id so it can be 
+  # referred by other stages
   - from:
       source: "dockerhub"
       repository: "itisfoundation/sleeper"
     to:
       - destination: master
         repository: "simcore/services/comp/sleeper"
         tags: []
+    id: "i-run-before"  # eeded for the stage below
   
-  # because the previous stage already synced the images
-  # pulls all the tag 1.0.0 of sleeper from master syncs to staging (also in a different namespace)
+  # because the depends_on tag was added, this stage will wait for the specified
+  # sages to complete before starting
+  # Please note: if an error occurs in one of the depends_on stages execution
+  # still goes on. Error logs will always be displayed if something went wrong
+  # for easy debugging
```

Comment:
is that a wanted feature? if a dependency fails it still runs? I don't really see the point in doing this.

Reply:
This is intentional because it was easier to implement with the parallel image syncing. Due to the repeating usage of this program, it has little to no influence overall.